platform/jewel/samples/showcase/api-dump.txt

```diff
@@ -9,9 +48,8 @@ f:org.jetbrains.jewel.samples.showcase.components.ComposableSingletons$TooltipsK
 - f:getLambda$-1070103804$intellij_platform_jewel_samples_showcase():kotlin.jvm.functions.Function2
 - f:getLambda$-186585630$intellij_platform_jewel_samples_showcase():kotlin.jvm.functions.Function2
 - f:getLambda$-408485079$intellij_platform_jewel_samples_showcase():kotlin.jvm.functions.Function2
-f:org.jetbrains.jewel.samples.showcase.components.ShowcaseIcons$Components
-- f:getTypography():org.jetbrains.jewel.ui.icon.PathIconKey
```

Comment:
This is the showcase module, which is not published. We don't care about API stability :)